ui/src/alert-configuration/ConfigureAlerting.test.tsx

```diff
@@ -0,0 +1,1275 @@
+      const result = [];
+      // return success status
+      return res(ctx.json(result));
+    },
+  ),
+
+  rest.get(
+    `http://localhost/api/kubernetes/api/v1/namespaces/metalk8s-monitoring/configmaps/metalk8s-alertmanager-config`,
+    (req, res, ctx) => {
+      const test = {
+        kind: 'ConfigMap',
+        apiVersion: 'v1',
+        metadata: {
+          name: 'metalk8s-alertmanager-config',
+          namespace: 'metalk8s-monitoring',
+          uid: 'xxx-yyy-zzz',
+          resourceVersion: '0',
+          creationTimestamp: '2023-03-23T16:45:41Z',
+          labels: {},
+          managedFields: [],
+        },
+        data: {
+          'config.yaml':
+            'apiVersion: addons.metalk8s.scality.com\nkind: AlertmanagerConfig\nspec:\n  notification:\n    config:\n',
+        },
+      };
+      return res(ctx.json(test));
+    },
+  ),
+
+  rest.get(
+    `http://localhost/api/kubernetes/api/v1/namespaces/metalk8s-monitoring/pods/alertmanager-prometheus-operator-alertmanager-0/log`,
+    (req, res, ctx) => {
+      const logs = '';
+      return res(ctx.text(logs));
+    },
+  ),
+
+  rest.get('http://localhost/brand/email.html', (req, res, ctx) => {
+    const result = 'TEMPLATE DELETED';
+    return res(ctx.text(result));
+  }),
+
+  rest.get('http://localhost/logo.svg', (req, res, ctx) => {
+    const result = 'LOGO';
+    return res(ctx.text(result));
+  }),
+
+  rest.patch(
+    'http://localhost/api/kubernetes/api/v1/namespaces/metalk8s-monitoring/configmaps/metalk8s-alertmanager-config',
+    (req, res, ctx) => {
+      const body = req.body;
+      patchAlertmanagerConfig(body);
+      return res(ctx.json({}));
+    },
+  ),
+
+  rest.post('http://localhost/api/salt/login', (req, res, ctx) => {
+    const body = req.body;
+    saltLoginRequest(body);
+
+    return res(
+      ctx.json({
+        return: [
+          {
+            token: 'xxx-yyy-zzzz-token',
+          },
+        ],
+      }),
+    );
+  }),
+
+  rest.post('http://localhost/api/salt', (req, res, ctx) => {
+    return res(ctx.json({}));
+  }),
+
+  rest.get('http://localhost/api/kubernetes/api/v1/nodes', (req, res, ctx) => {
+    const result = {
+      items: [
+        {
+          metadata: {
+            name: 'node1',
+            labels: {
+              'metalk8s.scality.com/version': '125.0.0',
+            },
+          },
+        },
+      ],
+    };
+    return res(ctx.json(result));
+  }),
+
+  rest.get(
+    'http://localhost/api/kubernetes/api/v1/namespaces/metalk8s-monitoring/pods',
+    (req, res, ctx) => {
+      const result = {
+        items: [
+          {
+            metadata: {
+              name: 'pods',
+            },
+          },
+        ],
+      };
+      return res(ctx.json(result));
+    },
+  ),
+
+  rest.post(
+    'http://localhost/api/alertmanager/api/v1/alerts',
+    (req, res, ctx) => {
+      const result = {
+        status: 'success',
+      };
+      return res(ctx.json(result));
+    },
+  ),
```

Comment:
this seems kind of a duplicate of https://github.com/scality/metalk8s/pull/4078/files#diff-d39d9dbe5273942a1dccfb4560b204df6206b32c59bf0f3b3c559d0f74fe8bd7R79 line 79